litert/vendors/intel_openvino/.clang-format

```diff
@@ -0,0 +1,150 @@
+---
```

Comment:
strange to have separate code style for openvino. I think it should be common for the whole litert

Reply:
+1